xamarin/Xamarin.Forms
# Fixes Bug with TabbedPage pushed onto Shell

### Description of Change ###

Fixes an issue when pushing a TabbedPage onto a Shell. 

On iOS the `TabbedRenderer` did not propagate size updates to the children, which resulting in them being hidden.

On Android the `PageRenderer` did not set the default background for the children of the `TabbedPage` and since the `TabbedPage` does not have a background the shell was still visible.

### Issues Resolved ### 
- fixes #6384

### API Changes ###
 
 None

### Platforms Affected ### 
- iOS
- Android

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 
![android-bug](https://user-images.githubusercontent.com/9119485/64948684-da7f6280-d877-11e9-8315-4f2cace225e9.png)
![android-fix](https://user-images.githubusercontent.com/9119485/64948685-da7f6280-d877-11e9-8e7c-a1077455dd83.png)
![iphone-bug](https://user-images.githubusercontent.com/9119485/64948689-dd7a5300-d877-11e9-9460-23efd927253f.png)
![iphone-fixed](https://user-images.githubusercontent.com/9119485/64948690-de12e980-d877-11e9-8aca-c84632d6722e.png)


### Testing Procedure ###
The `Github6384` test page shows this fix.
*Note: The UI Test does only catch the bug on iOS and not on Android. I wasn't able to check the background color of a nativ view in the UI Test.*

### PR Checklist ###
